src/app/malaria-101/malaria-101.component.scss

```diff
@@ -0,0 +1,228 @@
+@import "../styles/main";
+
+video{
```

Comment:
1. Made changes to the file spacing. 
2. Can you please help me identify the variables?
3. No I am not using it anywhere, have removed it now.
4. I am not sure if I am using img styling in any other file, but this img styling is specific to this activity.
5. Here is the video for the responsiveness @yukie1412 
https://drive.google.com/file/d/0B7Hnx8cG8DNBUWhVbUxMVmt3LXM/view?usp=sharing